dist/es-module-shims.js

```diff
@@ -3,7 +3,14 @@
   'use strict';
 
   let baseUrl;
-  if (typeof location !== 'undefined') {
+
+  if (typeof document !== 'undefined') {
+    const baseEl = document.head.querySelector('base[href]');
```

Comment:
To be incredibly pedantic, I think we can save a byte or two by making this `document.querySelector('base[href]')`.